sstables/sstables_manager.hh

```diff
@@ -96,6 +96,11 @@ private:
     list_type _active;
     list_type _undergoing_close;
 
+    // Total memory used by components of sstables in _active list
+    uint64_t _total_memory_used_by_components{0};
+    // Maximum allowed memory for all components across all sstables.
+    uint64_t _max_allowed_memory_for_components;
+
```

Comment:
size_t etc.

Reply:
_max_allowed_memory shoudl be in a different patch, no?

Reply:
Ack. Updated.